code/datums/components/wanderlust.dm

```diff
@@ -0,0 +1,33 @@
+/datum/component/wanderlust
```

Comment:
make sure the component name is something that actually tell what it is on first look, instead of having to figure out what "wanderlust" even means